src/Draco.Compiler/Internal/Symbols/Metadata/MetadataStaticClassSymbol.cs

```diff
@@ -57,6 +67,35 @@ private ImmutableArray<Symbol> BuildMembers()
             result.Add(methodSym);
         }
 
+        // Fields
+        foreach (var fieldHandle in this.typeDefinition.GetFields())
+        {
+            var fieldDef = this.MetadataReader.GetFieldDefinition(fieldHandle);
+            // Skip fields with special name
+            if (fieldDef.Attributes.HasFlag(FieldAttributes.SpecialName)) continue;
+            // Skip non-public fields
+            if (!fieldDef.Attributes.HasFlag(FieldAttributes.Public)) continue;
+            // Skip non-static fields
+            if (!fieldDef.Attributes.HasFlag(FieldAttributes.Static)) continue;
+            var fieldSym = new MetadataFieldSymbol(
+                containingSymbol: this,
+                fieldDefinition: fieldDef);
+            result.Add(fieldSym);
+        }
+
+        var defaultName = MetadataSymbol.GetDefaultMemberAttributeName(this.typeDefinition, this.DeclaringCompilation!, this.MetadataReader);
```

Comment:
Could this become a lazy property for this and regular class? Then properties would just walk up and ask the container about it.